blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md

```diff
@@ -0,0 +1,38 @@
+---
+title: First Week Wrap Up! Team potatoCode
+layout: post
+created_at: Sat Jul 07 2018
+permalink: 2018-07-07-first-official-contribution-team-potatoCode
```

Comment:
Well, the format for permalink is blog/<file_name>. What I meant was to just change the date. Sorry if that wasn't clear.